### ISSUE TITLE
UpBar - buttons not visible

### ISSUE THREAD

**Issue by rlemon** [rlemon/UserScripts#1]
the buttons at the top of the chat are not visible to the user because they are blocked by the UpBar... no ideas as of yet.

**Comment by amithkk** [comment]
hey why not make it dissapear when it reaches the top?
_You Could do that_

**Comment by rlemon** [comment]
Already done :) scroll to the top and see
